packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -797,6 +797,7 @@ export type CallCompositeOptions = {
     spotlight?: {
         hideSpotlightButtons?: boolean;
     };
+    skipMicCheck?: boolean;
```

Comment:
This name needs to be more specific, it doesn't say what microphone checks are skipped. We should also try to always use `microphone` instead of `mic`. Consider:
```suggestion
    joinCallOptions: {
        doNotBlockOnMicrophoneAccess: boolean;
    }
```
There are suggestions on your original design doc as well.

Reply:
Of these options I think I like the last one best as it gives more control to Contoso